pkg/cli/image/extract/extract.go

```diff
@@ -188,7 +188,7 @@ type Mapping struct {
 	ConditionFn func(m *Mapping, dgst digest.Digest, imageConfig *dockerv1client.DockerImageConfig) (bool, error)
 }
 
-func parseMappings(images, paths, files []string, requireEmpty bool) ([]Mapping, error) {
+func parseMappings(images, paths, files []string, fileDir string, requireEmpty bool) ([]Mapping, error) {
```

Comment:
Hrm, dir should be applied later (IIRC, but I might be wrong), so I'm not even sure this is the correct code path.  Will have to review.

Reply:
FileDir has nothing to do with extract.   --dir controls "the directory on disk where image sources of type file will be located", not "the path that things are etxracted to".

/hold

Reply:
FileDir is already used https://github.com/openshift/oc/pull/685/files#diff-77442d06befb7d25d1cf6e73620544af7c772eac2d2dd6f457bb3062117350b9L323

If we need to update the help text to make this clear, we should so with this PR.

Reply:
So, there is no flag to control where things are extracted to? Always to the current directory?

Reply:
maybe we could similarly how `oc adm release mirror` has:
```
--to-dir='': A directory to export images to.
```
?

Reply:
yes, I've added to-dir to the new PR, same code as this but new flag --to-dir rather than --dir